src/resources/V2/Billing.ts

```diff
@@ -1,16 +0,0 @@
-// File generated from our OpenAPI spec
-
-import {StripeResource} from '../../StripeResource.js';
-import {MeterEventSession} from './Billing/MeterEventSession.js';
-import {MeterEventAdjustments} from './Billing/MeterEventAdjustments.js';
-import {MeterEventStream} from './Billing/MeterEventStream.js';
-import {MeterEvents} from './Billing/MeterEvents.js';
-export const Billing = StripeResource.extend({
-  constructor: function(...args: any) {
-    StripeResource.apply(this, args);
-    this.meterEventSession = new MeterEventSession(...args);
```

Comment:
In this version, `Billing.meterEventSession` is an instance of `MeterEventSession`, but in your updated version it's the `MeterEventSession` itself. Is that intentional? does it change the way users can use the `v2` object?

Reply:
good question! are you referring to this code from src/resources.ts ?
```typescript
export const V2 = resourceNamespace('v2', {
  Billing: resourceNamespace('billing', {
    MeterEventAdjustments: V2BillingMeterEventAdjustments,
    MeterEventSession: V2BillingMeterEventSession,
    MeterEventStream: V2BillingMeterEventStream,
    MeterEvents: V2BillingMeterEvents,
  }),
  Core: resourceNamespace('core', {
    EventDestinations: V2CoreEventDestinations,
    Events: V2CoreEvents,
  }),
});
```

The `resourceNamespace` function accepts resources (aka sub services in this context) as an object and automatically camel cases the keys.  Apologies, I thought I had a test script in a comment here but I think I only included it in the codegen PR.  Here's the script I tested with:
```typescript
const apiKey = ...
const customerId = ...
const meterObjectId = ...

(async function() {
  const client = new Stripe(apiKey);
  const meters = await client.v2.billing.meterEvents.create({
    event_name: 'api_requests',
    payload: {
      stripe_customer_id: customerId,
      value: '27',
    },
  });
  console.log(meters);
  const events = await client.v2.core.events.list({
    object_id: meterObjectId,
  });
  console.log(events);
  const dests = await client.v2.core.eventDestinations.list();
  console.log(dests);
})();
```

Reply:
Although, as noted above though, it does technically break if a user uses that Stripe.V2.BillingResource type to hold the empty resource we're now deleting.

Reply:
Ok, I think that makes sense. This seems like a good improvement!

Reply:
Thanks!  Resolving this conversation and merging.